tests/pytest/test.py

```diff
@@ -1280,6 +1280,123 @@ def testFilterPrecedence(env):
+                
+                res = r.execute_command('JSON.NUMMULTBY', name, '$.{}'.format(k), '2')
+                if check_num_mult:
+                    r.assertAlmostEqual(json.loads(res)[0], v*2, epsilon, message=res)
+                r.execute_command('JSON.NUMMULTBY', name, '$.{}'.format(k), '9223372036854775808')
+                r.execute_command('JSON.NUMMULTBY', name, '$.{}'.format(k), '18446744073709551615')
+                r.execute_command('JSON.NUMMULTBY', name, '$.{}'.format(k), '18446744073709551616')
+                r.execute_command('JSON.NUMMULTBY', name, '$.{}'.format(k), '-9223372036854775808')
+                r.execute_command('JSON.NUMMULTBY', name, '$.{}'.format(k), '-18446744073709551615')
+                r.execute_command('JSON.NUMMULTBY', name, '$.{}'.format(k), '-18446744073709551616')
+            except Exception as e:
+                r.assertTrue('u64 is not supported' in str(e) or 'result is not a number' in str(e))
+            arr.append(v)
+        # Check no crash using values in a filter
+        r.expect('JSON.SET', 'arr', '$', json.dumps(arr)).ok()
+        res = r.execute_command('JSON.GET', 'arr', '$[?(@!=0)]')
+        for res_i, arr_i in zip(json.loads(res), arr):
+            r.assertAlmostEqual(res_i, arr_i, epsilon, message=res)
+        # Check no crash with other commands 
+        r.expect('JSON.ARRINDEX', 'arr', '$', '10').equal([-1])
+        res = r.execute_command('JSON.RESP', 'arr')
+        for res_i, arr_i in zip(res[1:], arr):
+            if type(res_i) is int:
+                # Integers greater then max i64 are returned as max i64 by RESP
+                r.assertAlmostEqual(res_i, arr_i if arr_i <= sys.maxsize else sys.maxsize, epsilon, message=res)
+            elif type(res_i) is str:
+                # Double are returned as strings
+                r.assertAlmostEqual(float(res_i), arr_i, epsilon, message=res)
+        for k, v in iter(obj.items()):            
+            r.execute_command('JSON.CLEAR', name, '$.{}'.format(k))
+
+    check_object_values(doc_int_ok, 'doc_int_ok', 0)
+    check_object_values(doc_int_ok_overflow_mult, 'doc_int_ok_overflow_mult', 0, True, False)
+    check_object_values(doc_int_ok_overflow_both, 'doc_int_ok_overflow_both', 0, False, False)
+    
+    check_object_values(doc_float_ok, 'doc_float_ok', sys.float_info.epsilon)
+    check_object_values(doc_float_ok_overflow_mult, 'doc_float_ok_overflow_mult', 0, True, False)
+
+    check_object_values(doc_float_ok_2_overflow_mult, 'doc_float_ok_2_overflow_mult', sys.float_info.epsilon, True, False)
+    
+    # Not using json.dumps with out-of-range values here (would be converted to a string representation such as 'Infinity')
+    r.expect('JSON.SET', 'doc_bad_values', '$', '{}').ok()
+    for k, v in iter(doc_bad_values.items()):
+        r.assertTrue(True, message='SET {}={}'.format(k, v))
+        r.expect('JSON.SET', 'doc_bad_values', '$.{}'.format(k), '{}'.format(v)).error()
```

Comment:
```suggestion
        r.expect('JSON.SET', 'doc_bad_values', '$.{}'.format(k), '{}'.format(v)).error().contians(<some substring from the expected error>)
```

Reply:
This errors are JSON parse errors, such as with these commands
```
"JSON.SET" "doc_bad_values" "$.beyond_max_f64" "inf"
"JSON.SET" "doc_bad_values" "$.below_min_f64" "-inf"
```
these error messages are issued:
`expected value at line 1 column 1`
`invalid number at line 1 column 2`